services/Player.js

```diff
@@ -0,0 +1,80 @@
+import db from '../db'
+
+class Player {
+  pickRandomlySong() {
+    return db
+      .select('id')
+      .from('songs')
+      .limit(10)
+      .map(s => s.id)
+  }
+
+  getSongById(songId) {
+    return db('songs')
+      .join('artists', 'artists.id', '=', 'songs.artist_id')
+      .join('albums', 'albums.id', '=', 'songs.album_id')
+      .select(
+        'songs.id',
+        'songs.title',
+        'albums.title as albumTitle',
+        'albums.artworkPath',
+        'artists.name as artistName',
+        'songs.duration',
+        'songs.path',
+        'songs.albumOrder',
+        'songs.plays',
+        'songs.artist_id',
+        'songs.album_id',
+        'songs.genre_id'
+      )
+      .where('songs.id', songId)
+      .first()
+  }
+
+  incrementSongPlays(songId) {
+    return db('songs')
+      .where('id', songId)
+      .increment({ plays: 1 })
+  }
+
+  getAlbums() {
+    return db
+      .select('*')
+      .from('albums')
+      .orderBy('RAND()')
+      .limit(10)
+  }
+
+  getAlbumById(id) {
+    return db
+      .select('*')
+      .from('albums')
+      .where('id', id)
+      .first()
+  }
+
+  getArtistByAlbum(artist_id) {
```

Comment:
This method is not properly named.
You are retrieving Artist by `artist_id`, not `byAlbum`